src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -81,8 +82,13 @@ class ReportActionItemMessageEdit extends React.Component {
         this.emojiButtonID = 'emojiButton';
         this.messageEditInput = 'messageEditInput';
 
-        const parser = new ExpensiMark();
-        const draftMessage = parser.htmlToMarkdown(this.props.draftMessage).trim();
+        let draftMessage;
+        if (this.props.draftMessage === this.props.action.message[0].html) {
```

Comment:
@eh2077 Could you add a comment explaining why we doing this and what issue we trying to solve?

Reply:
Added comments